src/flint/test/test_all.py

```diff
@@ -2021,8 +2023,8 @@ def test_fmpz_mod_poly():
 
     # Arithmetic
     p_sml = 163
-    p_med = 2**127 - 1
-    p_big = 2**255 - 19
+    p_med = 167
+    p_big = 173
 
     F_sml = fmpz_mod_ctx(p_sml)
     F_med = fmpz_mod_ctx(p_med)
```

Comment:
This PR modifies `test_fmpz_mod` to avoid the large integers.

Reply:
Would you like me to try with `p_big = 2**255 - 19`?